### PR TITLE
NiRi3

### PR DESCRIPTION
**Pull request by svelamuri26**

### CODE REVIEW

**Comment by MenilV** [comment]
Let's please add Jira ticket number in the commit messages in the following format:

"NIRI-27 Add Cart Item module"

and also reduce these PRs since this is too big and has multiple tickets in it.

File: src/main/java/com/example/NiRi/SwaggerConfig.java
Comment (MenilV) [inline src/main/java/com/example/NiRi/SwaggerConfig.java:1]: do we need this? is this the file causing issues?

File: src/main/java/com/example/NiRi/modules/User.java [hunk 1]
```diff
@@ -1,6 +1,9 @@
 package com.example.NiRi.modules;
 
+<<<<<<< HEAD
+=======
 
+>>>>>>> origin/main
```
Comment (MenilV) [inline src/main/java/com/example/NiRi/modules/User.java]: this is a conflict, please resolve it.

**Review by MenilV — CHANGES_REQUESTED** [review]
A few things to check but overall good.

File: src/main/java/com/example/NiRi/SwaggerConfig.java
Comment (svelamuri26) [reply src/main/java/com/example/NiRi/SwaggerConfig.java:1]: I just forgot to delete it. I did it now.

File: src/main/java/com/example/NiRi/modules/User.java (hunk 1 above)
Comment (svelamuri26) [reply src/main/java/com/example/NiRi/modules/User.java]: resolved and pushed again.

**Comment by svelamuri26** [comment]
Thank you for the feedback.

**Review by MenilV — APPROVED** [review]

**Pull request merged**